portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationAssigningRoles.testcase

```diff
@@ -853,11 +843,8 @@ definition {
 		}
 	}
 
-	@description = "LPS-167023 Given a user with Segments permissions
-	And with View Control Panel Menu permission
-	When the user clicks on Applications Menu > Control Panel > Segment
-	Then the user can access to the Segments page"
-	@priority = "5"
+	@description = "LPS-167023: Validate if the user with View Control Panel Menu permission can access to the Segments page"
```

Comment:
For the description, just put the original description but in one line

Reply:
done!